disnake/app_commands.py

```diff
@@ -102,6 +102,10 @@ def __eq__(self, other) -> bool:
     def to_dict(self) -> Dict[str, ChoiceValue]:
         return {"name": self.name, "value": self.value}
 
+    @classmethod
+    def from_dict(cls, data):
+        return OptionChoice(name=data.get("name"), value=data.get("value"))
```

Comment:
```suggestion
        return OptionChoice(name=data["name"], value=data["value"])
```
These aren't optional and should always exist